Can now pop balloons and quack ducks in title screen. (#9614)
OpenRCT2/OpenRCT2: distribution/changelog.txt

------------------------------------------------------------------------
- Feature: [#7648] "Enable all drawable track pieces" now enables more pieces for the Twister, Vertical and Air Powered Vertical coasters.
- Feature: [#9029] Open doors with the tile inspector.

> - Feature: [#9614] Can now pop balloons and quack ducks in title screen.

Reviewer: ```suggestion
- Feature: [#9614] Allow popping balloons and quacking ducks in the title screen.
```

- Feature: [#6553] Android version now runs in full screen.
- Feature: [#7865] Transport rides can now be synchronised.
- Feature: [#9073] Shortcut keys for the Tile Inspector.

> - Feature: [#9614] Can now pop balloons and quack ducks in title screen.

Reviewer: ```suggestion
- Feature: [#9614] Allow interaction with balloons and ducks in the title screen.
```
Also needs to be put under the appropriate version.
